src/big_widgets/GitQlient.cpp

```diff
@@ -110,67 +102,82 @@ void GitQlient::setRepositories(const QStringList &repositories)
       addRepoTab(repo);
 }
 
-void GitQlient::setArgumentsPostInit(const QStringList &arguments)
+bool GitQlient::setArgumentsPostInit(const QStringList &arguments)
 {
    QLog_Info("UI", QString("External call with the params {%1}").arg(arguments.join(",")));
 
-   const auto repos = parseArguments(arguments);
-
-   setRepositories(repos);
+   QStringList repos;
+   bool ret = parseArguments(arguments, &repos);
+   if (ret) {
+      setRepositories(repos);
+   }
+   return ret;
 }
 
-QStringList GitQlient::parseArguments(const QStringList &arguments)
+bool GitQlient::parseArguments(const QStringList &arguments, QStringList *repos)
 {
-
-   LogLevel logLevel;
+   bool ret = true;
    GitQlientSettings settings;
-
 #ifdef DEBUG
-   logLevel = LogLevel::Trace;
+   LogLevel logLevel = LogLevel::Trace;
 #else
-   logLevel = static_cast<LogLevel>(settings.globalValue("logsLevel", static_cast<int>(LogLevel::Warning)).toInt());
+   LogLevel logLevel = static_cast<LogLevel>(settings.globalValue("logsLevel", static_cast<int>(LogLevel::Warning)).toInt());
 #endif
+   bool areLogsDisabled = settings.globalValue("logsDisabled", true).toBool();
 
-   if (arguments.contains("-noLog") || settings.globalValue("logsDisabled", true).toBool())
-      QLoggerManager::getInstance()->pause();
-   else
-      QLoggerManager::getInstance()->overwriteLogLevel(logLevel);
+   QCommandLineParser parser;
+   parser.setApplicationDescription(tr("Graphical Git client"));
```

Comment:
I would use the header from the readme if possible:

`GitQlient: Multi-platform Git client written with Qt`

Reply:
Done